src/iarc7_motion/iarc_tasks/go_to_roomba_task.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python
+
+import math
+import rospy
+import tf2_ros
+import tf2_geometry_msgs
+import threading
+
+from geometry_msgs.msg import TwistStamped
+from geometry_msgs.msg import PointStamped
+from geometry_msgs.msg import Point
+from iarc7_msgs.msg import OdometryArray
+from nav_msgs.msg import Odometry
+from geometry_msgs.msg import Vector3Stamped
+from geometry_msgs.msg import Vector3
+
+from .abstract_task import AbstractTask
+from iarc_tasks.task_states import (TaskRunning,
+                                    TaskDone,
+                                    TaskCanceled,
+                                    TaskAborted,
+                                    TaskFailed)
+from iarc_tasks.task_commands import (VelocityCommand,
+                                      ArmCommand,
+                                      NopCommand)
+
+from height_holder import HeightHolder
+from height_settings_checker import HeightSettingsChecker
+from acceleration_limiter import AccelerationLimiter
+
+class GoToObjectTaskState(object):
```

Comment:
can you call this `GoToRoombaTaskState`?

Reply:
Fixed